.drone.star

```diff
@@ -534,6 +534,38 @@ def changelog(ctx):
 
 	return pipelines
 
+def unitTests():
+	return [{
+		'kind': 'pipeline',
+		'type': 'docker',
+		'name': 'unit-tests',
+		'steps': [
+			{
+				'name': 'install-deps',
+				'image': 'owncloudci/nodejs:12',
```

Comment:
do we care about running the tests on different node version?

Reply:
I was thinking about this and TBH I'm not sure. We're limiting node version 10 - 13. This means 4 potential different versions of Node. We do not have an image for all of them though.